sklearn/ensemble/gradient_boosting.py

```diff
@@ -271,13 +371,19 @@ def predict(self, X):
 
 @deprecated("Using ZeroEstimator is deprecated in version "
             "0.21 and will be removed in version 0.23.")
-class ZeroEstimator:
+class ZeroEstimator(object):
```

Comment:
please revert this change (all classes are objects in python anyway).

Reply:
@adrinjalali Fixed.